### PR TITLE
[Typo] Optimize some code comments

### PR DESCRIPTION
**Pull request by xlwh**
# Proposed changes

Issue Number: close #xxx

## Problem Summary:

Describe the overview of changes.

## Checklist(Required)

1. Does it affect the original behavior: No
2. Has unit tests been added: No Need
3. Has document been added or modified: No Need
4. Does it need to update dependencies: No
5. Are there any changes that cannot be rolled back: No

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by jackwener — DISMISSED** [review]
Thanks your contribution! ❤
I compared with [google cpp style](https://google.github.io/styleguide/cppguide.html#Comment_Style). It's OK.

LGTM

File: be/src/olap/version_graph.cpp [hunk 1]
```diff
@@ -57,7 +57,7 @@ void TimestampedVersionTracker::construct_versioned_tracker(
     _next_path_id = 1;
     _construct_versioned_tracker(rs_metas);
 
-    // init _stale_version_path_map
+    // Init '_stale_version_path_map'.
```
Comment (jackwener) [inline be/src/olap/version_graph.cpp:60]:
```suggestion
    // Init `_stale_version_path_map`.
```

Use  ` would be better. 

We can see `Function Comments` in `google cpp style`. [here](https://google.github.io/styleguide/cppguide.html#Comment_Style)

File: be/src/olap/version_graph.cpp [hunk 2]
```diff
@@ -68,15 +68,15 @@ void TimestampedVersionTracker::_init_stale_version_path_map(
...
     std::list<RowsetMetaSharedPtr> sorted_stale_metas;
     for (auto& rs : stale_metas) {
         sorted_stale_metas.emplace_back(rs);
     }
 
-    // 1. Sort the existing rowsets by version in ascending order
+    // 1. sort the existing rowsets by version in ascending order.
     sorted_stale_metas.sort([](const RowsetMetaSharedPtr& a, const RowsetMetaSharedPtr& b) {
-        // compare by version diff between version.first and version.second
+        // Compare by version diff between 'version.first' and 'version.second'.
```
Comment (jackwener) [inline be/src/olap/version_graph.cpp:79]: ditto

File: be/src/olap/version_graph.cpp [hunk 3]
```diff
@@ -98,19 +98,19 @@ void TimestampedVersionTracker::_init_stale_version_path_map(
...
         add_version(stale_meta_version);
 
-        // 2.3 find the path
+        // 2.3 find the path.
         if (r) {
-            // add the path to _stale_version_path_map
+            // Add the path to '_stale_version_path_map'.
             add_stale_path_version(stale_path);
-            // remove stale_path from stale_map
+            // Remove 'stale_path' from 'stale_map'.
```
Comment (jackwener) [inline be/src/olap/version_graph.cpp:113]: ditto, and following I  won't list.

**Review by jackwener — APPROVED** [review]
LGTM. Thanks for your contributions.❤

**Review by hf200012 — APPROVED** [review]
LGTM

**Pull request merged**